ReactiveExample/ReactiveExample.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+    <PropertyGroup>
+        <OutputType>Exe</OutputType>
+        <TargetFramework>netcoreapp3.1</TargetFramework>
+    </PropertyGroup>
+    <ItemGroup>
+        <PackageReference Include="Pharmacist.MsBuild" Version="1.8.1" PrivateAssets="all" />
+        <PackageReference Include="Pharmacist.Common" Version="1.8.1" />
+        <PackageReference Include="Terminal.Gui" Version="0.90.3" />
```

Comment:
Since this will be part of the Terminal.Gui project, shouldn't this ref the csproj file?

Reply:
That way it stay limited do to the 0.90.3 version. It's bad. Good catch.

Reply:
The issue here is, that [Pharmacist.MSBuild](https://github.com/reactiveui/pharmacist#msbuild) can generate `.Events()` only for `.dll` assemblies and NuGet packages. So if we include `Terminal.Gui` via a `ProjectReference`, the tool won't work. Currently, Pharmacist is being rewritten [to use the new source generators API](https://github.com/reactivemarbles/ObservableEventsSourceGenerator), so hopefully, a new version will be out soon, that won't impose such limitations because the new source generators can work with any source file. We could declare the version attribute as `Version="*"` but this way it has a chance to break the CI build after a new NuGet package version is out, so decided to pin the version temporarily.

Another way of resolving this is to prepare a separate NuGet package e.g. `Terminal.Gui.Events` that will contain the generated observable wrappers (we are [using this approach](https://github.com/AvaloniaUI/Avalonia/blob/master/nukebuild/Build.cs#L188) in Avalonia 0.10). But after the new source generators-based Pharmacist is out, that package will become no longer required, so not sure if publishing a new NuGet makes sense now...

Reply:
Got it. 

How about this: We can now easily generate pre-release nuget packages for Terminal.Gui by simply updating the version info with a `-pre` indicator via `Terminal.Gui/Directory.build.props`. Could you build a pre-build tool (using `TT`?) to suck the `<version>xxx</version>` string from `Terminal.Gui/Directory.Build.props`?

Oh, that won't work... because the nuget package isn't published and available for several minutes after a push to master.

Hmmm.... 

I guess for now we'll just have to live with the risk that your sample will pull and older nuget package. Coordination will be needed. But it's not a big deal. 

For now, do update to use the `1.0.0-pre.1` nuget package.

Reply:
On it. Will update the version soon.